app/src/AppRouter.tsx

```diff
@@ -86,14 +96,18 @@ const AppRouter: React.FC = () => {
         </BaseLayout>
       </RouteWithTitle>
 
-      <RouteWithTitle path="/admin/resources" title={getTitle('Resources')}>
+      <RouteWithTitle path="/resources" title={getTitle('Resources')}>
         <BaseLayout>
-          <AuthenticatedRouteGuard>
-            <ResourcesPage />
-          </AuthenticatedRouteGuard>
+          <ResourcesPage />
         </BaseLayout>
       </RouteWithTitle>
 
+      <Route path="/login">
+        <UnAuthenticatedRouteGuard>
+          <LoginStub />
```

Comment:
We do have a logout page, might be worth turning this into a login page, and storing it alongside the logoutpage, just to keep them together.

Reply:
Also just to keep the approuter clean